shared/src/commonMain/kotlin/com/example/musicapp_kmp/decompose/ChartDetailsComponent.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.musicapp_kmp.decompose
```

Comment:
We don't need this file as it already exist. The package has been updated.

Reply:
Okay. I will refactor changes.